### PR TITLE
[SDK-1667] XPS Enhancements 

### PR DESCRIPTION
**Pull request by root-ansh-ct**
[SDK-1667] : XPS Enhancements
- removing duplicate request to create notification. instead, just raising push notification viewed event when notification arrrives in `XiaomiMessageReceiver#onNotificationMessageArrived`
- supporting `wzrk_pn_s` to suppress notifications

[SDK-1667]: https://wizrocket.atlassian.net/browse/SDK-1667?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by pranaywarke** [comment]
We identified new issues on unchanged lines of code. Navigate to the Amazon CodeGuru Reviewer console to view the recommendations to fix them.

File: clevertap-xps/src/main/java/com/clevertap/android/xps/XiaomiMessageReceiver.java [hunk 1]
```diff
@@ -20,7 +20,6 @@ public class XiaomiMessageReceiver extends PushMessageReceiver {
     public void onNotificationMessageArrived(Context context, MiPushMessage miPushMessage) {
         super.onNotificationMessageArrived(context, miPushMessage);
         Logger.d(LOG_TAG, XIAOMI_LOG_TAG + "onNotificationMessageArrived is called");
-        handler.createNotification(context, miPushMessage);
```
Comment (root-ansh-ct) [inline clevertap-xps/src/main/java/com/clevertap/android/xps/XiaomiMessageReceiver.java:23]: add push notification viewed

**Review by darshanclevertap — APPROVED** [review]

**Review by piyush-kukadiya — APPROVED** [review]
LGTM

**Review by MrPiNkY2022 — APPROVED** [review]
Go ahead let me know what y'all do tho

**Pull request merged**